src/main/java/aeneas/models/PieceFactory.java

```diff
@@ -5,11 +5,286 @@
  * @author Joseph Martin
  */
 public class PieceFactory {
-  static Piece[] pieceIndex = {
-    /* TODO: Fill in pieces here */
+  public static Piece[] pieces = {
```

Comment:
I don't think this should be public - we don't want these instances being given out, since things like rotating pieces will modify them, and if all the instances of a particular type of hexonimo refer to the same instances from this array, then they'll all change when rotate etc. Instead, I think we want to keep the makePiece function, which should clone the piece before returning it.